packages/connection-form/src/components/advanced-options-tabs/authentication-tab/authentication-gssapi.tsx

```diff
@@ -99,9 +144,45 @@ function AuthenticationGSSAPI({
               value: value,
             });
           }}
-          label={GSSAPI_SERVICE_REALM_LABEL}
+          data-testid="gssapi-service-realm-input"
+          label="Service Realm"
           value={serviceRealm || ''}
+          optional
+        />
+      </FormFieldContainer>
+      <FormFieldContainer>
+        <Checkbox
+          data-testid="gssapi-password-checkbox"
+          checked={showPassword}
+          label="Provide password directly"
```

Comment:
Maybe just say “Provide password”? “directly” sounds like Compass would otherwise infer the password somehow

Reply:
That's kinda what I wanted to imply, isn't it what we mean? Maybe there is a way to say "hey you don't normally need the password"

Reply:
My (possibly wrong) understanding is that if there is an existing kerberos authentication context, the kerberos library would not need to fully authenticate again and can instead just use that (in which case there is no password coming up at any point)

But tbh, I’m also not sure on what the best wording here is